models/allegro/test_allegro_discovery.py

```
@@ -0,0 +1,163 @@
+# %% this config is editable
+compile_path = "*.nequip.pt2"
+smoke_test = False  # True
+model_name = "allegro-0"
+task_type = Task.IS2RE
+ase_optimizer = "GOQN"  # faster than "FIRE" from tests, gives the same results; see SI of https:/doi.org/10.1088/2515-7655/ade916
+ase_filter: Literal["frechet", "exp"] = "frechet"  # recommended filter
+
+max_steps = 500
+force_max = 0.05  # Run until the forces are smaller than this in eV/A
+
+slurm_nodes = int(os.getenv("SLURM_NNODES", 1))
```

Ruff (0.12.2)

Invalid type for environment variable default; expected `str` or `None`

(PLW1508)